tests/Unit/Evolution/Systems/Cce/BoundaryData.py

```diff
@@ -0,0 +1,240 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import numpy as np
+import math
+
+
+def cartesian_to_angular_coordinates(cos_phi, cos_theta, sin_phi, sin_theta,
+                                     extraction_radius):
+    return np.array([cos_phi * sin_theta, sin_phi * sin_theta, cos_theta])
+
+
+def cartesian_to_angular_jacobian(cos_phi, cos_theta, sin_phi, sin_theta,
+                                  extraction_radius):
+    return np.array(
+        [[cos_phi * sin_theta, sin_phi * sin_theta, cos_theta],
```

Comment:
Did you run this through something like autopep8 or yapf? The formatting looks interesting, but I don't have enough experience to be able to look at python formatting and know what formatters do :P

Reply:
yep, this is what yapf does to this file. Almost makes clang-format's lambda manipulation look good.

Reply:
Haha yea... Though clang format 9 seems to have figured out some things with lambdas at least...